packages/ucast-prisma/src/adapter.ts

```diff
@@ -25,3 +33,31 @@ export function ucastToPrisma(
     },
   })(parsed);
 }
+
+export interface Filters {
+  query: Record<string, any>;
+  mask<T extends Record<string, any>>(item: T): T;
+}
+
+export class Adapter extends OPAClient {
+  async filters<In extends Input | ToInput>(
```

Comment:
ℹ️ I couldn't override the name "getFilters()" because the types don't match. I still think extending is the right call, because we'll inherit all the OPAClient methods.